.travis.yml

```diff
@@ -1,22 +1,21 @@
 language: php
 
-php:
-# - 5.3 # requires old distro, see below
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
-  - 7.3
-
 # lock distro so future defaults will not break the build
 dist: trusty
 
 matrix:
   include:
     - php: 5.3
       dist: precise
+      install: composer remove react/mysql --dev --no-interaction # do not install react/mysql example on legacy PHP
```

Comment:
Leftover unrelated to this PR? Otherwise LGTM :+1:

Reply:
Fixed it :+1: